packages/atomic/src/pages/examples/suggestions.html

```diff
@@ -77,16 +77,64 @@
     </script>
   </head>
   <body>
-    <atomic-search-interface>
-      <atomic-search-layout>
+    <atomic-search-interface search-hub="MainSearch">
+      <atomic-search-layout mobile-breakpoint="700px">
+        <atomic-layout-section section="facets">
+          <atomic-facet-manager>
+            <atomic-facet-manager
+              facet-id="ec_rating_range"
+              field="ec_rating"
+              label="Rating Range"
+              number-of-intervals="5"
+              heading-level="2"
+            >
+            </atomic-facet-manager>
+          </atomic-facet-manager>
+        </atomic-layout-section>
         <atomic-layout-section section="search">
           <atomic-search-box suggestion-timeout="1000">
             <atomic-did-you-mean></atomic-did-you-mean>
             <atomic-search-box-recent-queries icon="assets://xml.svg"> </atomic-search-box-recent-queries>
-            <atomic-search-box-instant-results>
+            <atomic-search-box-instant-results image-size="small">
```

Comment:
can we remove the `<atomic-did-you-mean></atomic-did-you-mean>` and put back query suggestions here?

Reply:
line 96

Reply:
sure

Reply:
actually, why do you want to change it? can we leave both?